JSDemos/Demos/Form/Validation/Angular/app/app.component.html

```diff
@@ -75,6 +77,18 @@
         >
         </dxi-validation-rule>
       </dxi-item>
+      <dxi-item
+        dataField="VacationDates"
+        editorType="dxDateRangeBox"
+        [editorOptions]="{
```

Comment:
let's move it to a variable

Reply:
ok, i'll do it in my next pr with netcore demos since ga tasks take a really long time